lib/src/local_working_copy.rs

```diff
@@ -1565,18 +1610,36 @@ impl FileSnapshotter<'_> {
                 .cloned()
                 .flatten()
                 .unwrap_or_else(CopyId::placeholder);
+            let mut contents = vec![];
+            let file =
+                File::options()
+                    .read(true)
+                    .open(disk_path)
+                    .map_err(|err| SnapshotError::Other {
+                        message: format!("failed to open the file {}", disk_path.display()),
```

Comment:
nit: our convention of error message is "Failed .."

Reply:
Done.